packages/kit/src/components/Hardware/Hardware.tsx

```
@@ -194,6 +194,66 @@ export function EnterPinOnDevice({
+        );
+      case EDeviceType.Touch:
+        return import(
+          '@onekeyhq/kit/assets/animations/enter-hidden-wallet-pin-touch.json'
+        );
+      case EDeviceType.Pro:
+        return import(
+          '@onekeyhq/kit/assets/animations/enter-hidden-wallet-pin-pro-dark.json'
+        );
+      default:
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-case-declarations
+        const checkType = deviceType;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)